Compilation fails on FreeBSD

```
FreeBSD% cmake ..
-- C++11 regular expressions OK
-- libmagic version: 5.41
-- Found LibSndfile: /usr/local/lib/libsndfile.so
-- Found ucommon includes:      /usr/local/include/ucommon/ucommon.h
-- Found ucommon library:       /usr/local/lib/libucommon.so;/usr/local/lib/libusecure.so
-- Found commoncpp includes:    /usr/local/include/commoncpp/config.h
-- Found commoncpp library:     /usr/local/lib/libcommoncpp.so
-- Found ccrtp includes:        /usr/local/include/ccrtp/rtp.h
-- Found ccrtp library: /usr/local/lib/libccrtp.so
-- libasound OK
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/twinkle/make
FreeBSD% make
[  0%] Building CXX object src/CMakeFiles/libtwinkle.dir/abstract_dialog.cpp.o
In file included from /tmp/twinkle/src/abstract_dialog.cpp:21:
In file included from /tmp/twinkle/src/abstract_dialog.h:30:
In file included from /tmp/twinkle/src/client_request.h:26:
In file included from /tmp/twinkle/src/redirect.h:22:
In file included from /tmp/twinkle/src/user.h:28:
In file included from /tmp/twinkle/src/sys_settings.h:24:
In file included from /tmp/twinkle/src/parser/sip_message.h:90:
/tmp/twinkle/src/parser/sip_body.h:22:10: fatal error: 'commoncpp/config.h' file not found
#include <commoncpp/config.h>
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.                                                                               
make[2]: *** [src/CMakeFiles/libtwinkle.dir/build.make:76: src/CMakeFiles/libtwinkle.dir/abstract_dialog.cpp.o] Fehler 1
make[1]: *** [CMakeFiles/Makefile2:352: src/CMakeFiles/libtwinkle.dir/all] Fehler 2
make: *** [Makefile:136: all] Fehler 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,6 +50,16 @@ find_package(Ucommon REQUIRED)
 find_package(Commoncpp REQUIRED)
 find_package(Ccrtp REQUIRED)
 
+include_directories(
+	${LIBXML2_INCLUDE_DIR}
+	${LibMagic_INCLUDE_DIR}
+	${LIBSNDFILE_INCLUDE_DIR}
+	${Readline_INCLUDE_DIR}
+	${UCOMMON_INCLUDE_DIR}
+	${COMMONCPP_INCLUDE_DIR}
+	${CCRTP_INCLUDE_DIR}
+)
+
 if (WITH_QT5)
 	find_package(Qt5Widgets REQUIRED)
 	find_package(Qt5LinguistTools REQUIRED)
@@ -61,6 +71,7 @@ if (WITH_QT5)
 	if (WITH_DBUS)
 		find_package(Qt5DBus REQUIRED)
 		set(HAVE_DBUS TRUE)
+		include_directories(${Qt5DBus_INCLUDES})
 	endif (WITH_DBUS)
 
 	if (WITH_AKONADI)
@@ -68,10 +79,9 @@ if (WITH_QT5)
 		find_package(KF5Contacts REQUIRED)
 		message(STATUS "Akonadi OK")
 		set(HAVE_AKONADI TRUE)
+		include_directories(${KF5Akonadi_INCLUDES} ${KF5Contacts_INCLUDES})
 	endif (WITH_AKONADI)
 endif (WITH_QT5)
-
-include_directories(${LIBXML2_INCLUDE_DIR})
 
 if (WITH_ALSA)
 	find_package(ALSA)
@@ -79,6 +89,8 @@ if (WITH_ALSA)
 	if (ALSA_FOUND)
 		message(STATUS "libasound OK")
 		set(HAVE_LIBASOUND TRUE)
+
+		include_directories(${ALSA_INCLUDE_DIR})
 	else (ALSA_FOUND)
 		message(FATAL_ERROR "libasound not found!")
 	endif (ALSA_FOUND)

